runtime/src/main/java/org/corfudb/runtime/collections/streaming/StreamPollingScheduler.java

```diff
@@ -139,9 +141,10 @@ private void tick() {
         scheduler.submit(new Tick());
     }
 
-    public void addTask(@Nonnull StreamListener streamListener, @Nonnull String namespace,
-                        @Nonnull String streamTag, @Nonnull List<String> tablesOfInterest,
-                        long lastAddress, int bufferSize) {
+    public void addTask(@Nonnull StreamListener streamListener,
```

Comment:
A comment would be helpful (or example usage) here, specially this is a a public method.